azure/v13.0.0-beta1/release.yaml

```diff
@@ -19,40 +19,41 @@ spec:
     version: 1.5.0
   - componentVersion: 1.9.7
     name: kube-state-metrics
-    version: 1.2.0
+    version: 1.2.1
   - componentVersion: 0.3.3
     name: metrics-server
     version: 1.1.0
   - name: net-exporter
     version: 1.9.2
   - componentVersion: 1.0.1
     name: node-exporter
-    version: 1.4.2
+    version: 1.6.0
   components:
   - name: app-operator
     releaseOperatorDeploy: true
-    version: 2.1.1
+    version: 2.7.0
   - name: azure-operator
     releaseOperatorDeploy: true
-    version: 5.0.0-alpha4
+    version: 5.0.0-beta5
   - name: cert-operator
     reference: 0.1.0-2
     releaseOperatorDeploy: true
     version: 0.1.0
   - name: cluster-operator
     releaseOperatorDeploy: true
-    version: 0.23.16
+    version: 0.23.18
   - name: kubernetes
-    version: 1.18.10
+    version: 1.18.12
   - name: containerlinux
-    version: 2605.6.0
-  - name: coredns
-    version: 1.6.5
+    version: 2605.7.0
   - name: calico
     version: 3.15.3
   - name: etcd
-    version: 3.4.9
-  date: "2020-10-27T11:04:44Z"
+    version: 3.4.13
+  - name: chart-operator
```

Comment:
`chart-operator` is already listed as `app`. I guess it only needs to be added once?

Reply:
maybe, let's compare with latest AWS release

Reply:
removed